### PR TITLE
Project folders

### PR DESCRIPTION
**Pull request by mhmaldyb510**

### CODE REVIEW

**Comment by mhmaldyb510** [comment]
I add the text form field to the project

File: lib/main.dart [hunk 1]
```diff
@@ -1,125 +1,19 @@
 import 'package:flutter/material.dart';
 
-void main() {
-  runApp(const MyApp());
+void main(List<String> args) {
+  runApp(const SocialMedia());
```
Comment (Salint) [inline lib/main.dart:4]: Why args?

Comment (AhmedHelmy18) [reply lib/main.dart:4]: he write main and this function complete like this

Comment (Salint) [reply lib/main.dart:4]: Don't write useless code.

File: lib/main.dart [hunk 2]
```diff
@@ -1,125 +1,19 @@
 import 'package:flutter/material.dart';
 
-void main() {
-  runApp(const MyApp());
+void main(List<String> args) {
```
Comment (Salint) [inline lib/main.dart:3]: Remove unnecessary args.

**Review by Salint — CHANGES_REQUESTED** [review]

**Review by Salint — APPROVED** [review]

**Review by rofidarashed — APPROVED** [review]

**Pull request merged**